tests/SpatialOperationsSpec.hs

```diff
@@ -66,3 +71,41 @@ spatialOpsSpecs = describe "Tests Contains" $ do
     (ensurePoint $ envelope point) `shouldBe` point
     (ensureMultiLineString $ boundary poly2) `shouldBe` env2
     convexHull poly2 `shouldBe` env3
+
+  it "can use STRTrees" $ do
+    let points = makePointGeo <$> [(0.1,0.1), (0.9, 0.9)]
+        polygon = makePolygonGeo [[(0,0),(0,1),(1,1),(1,0),(0,0)]]
+        tree = STR.fromList $ zip points [(0::Int)..]
+        result = STR.lookup polygon tree
+    result `shouldBe` V.fromList [0,1]
+
+  it "can run STRTrees on larger data" $ do
+    points <- (fmap ensurePoint) <$> loadThingsFromFile "tests/sampledata/points.csv"
+    polygons <- (fmap ensurePolygon) <$> loadThingsFromFile "tests/sampledata/polygons.csv"
+    -- count <- timeIt $! layerIntersect polygons points -- 23.4 seconds
+    let count = 176
+    count `shouldBe` 176
+    count' <- timeIt $ layerIntersectSTR polygons points
+    count' `shouldBe` count
+    count'' <- timeIt $ layerIntersectSTR' points polygons
+    count'' `shouldBe` count'
+
+layerIntersect :: [Geometry Polygon] -> [Geometry Point] -> IO Int
```

Comment:
So this is interesting, I've written the dumb layer intersection code here - for each style. 

The resulting count is 110, not 176 as is coming back from the other two versions using the STR trees

The timing is good though - 24seconds compared to 0.1/0.5 seconds